src/adt/set.c

```diff
@@ -74,7 +74,7 @@ static int
 set_cmp(const void *a, const void *b)
 {
 
-	return (a > b) - (a < b);
+	return (a < b) - (a > b);
```

Comment:
Surely this was right as it was... don't we want to return +1 when `a > b` and -1 when `a < b`?